app/services/db-export/export-data-files.service.js

```diff
@@ -23,7 +23,7 @@ async function go (tableConvertedToCsv, tableName) {
     await fs.writeFile(_filenameWithPath(tableName), tableConvertedToCsv)
     global.GlobalNotifier.omg(`${tableName} exported successfully`)
 
-    return true
+    return _filenameWithPath(tableName)
```

Comment:
We are calling `_filenameWithPath()` twice and expecting to see the same value. There is also 'work' going on within that method that goes beyond adding some numbers or manipulating a string.

So, it would be better to put the result of `_filenameWithPath()` into a const first. Then pass it to `fs.writeFile()` and also return it as our service's result.